### DISCUSSION TITLE
como tener el bot 24/7 online?

### DISCUSSION THREAD

**Discussion by sanchesg1** [kaiserdj/Stickerworld#27] [Q&A] [answered]
Hola. Soy nuevo en esto, la verdad, es que quiero tener el bot 24/7 pero no quiero tener la pc prendida tood el tiempo, hay alguna manera con un servidor o algo? Gracias

**Comment by kaiserdj** [comment, marked as answer]
Si deseas tener el bot 24/7, lo unico que necesitas es tener un pc encendido todo el tiempo. Hay muchas maneras de hacerlo, dejando el tuyo propio, comprando otro, usar vps, usar servidores dedicados

**Reply by sanchesg1** [reply]
y como puedo hacerlo un vps? porque busque tutoriales de como ejecutar esta app en un vps o servidor y no se como. me ayudarias o pasarias un tutorial?

**Reply by kaiserdj** [reply]
Este es un ejemplo de como contratar un vps
https://www.youtube.com/watch?v=1kCNov9O_Ec

Simplemente necesitas un vps que tenga windows o windows server como sistema operativo, luego lo puedes usar como pc normal, por lo que instalas el stickerworld como un pc normal y ya esta
